src/plone/restapi/tests/http-examples/workingcopy_baseline_get.resp

```diff
@@ -2,83 +2,83 @@ HTTP/1.1 200 OK
-    }
-  }, 
-  "@id": "http://localhost:55001/plone/document", 
-  "@type": "Document", 
-  "UID": "SomeUUID000000000000000000000001", 
-  "allow_discussion": false, 
-  "contributors": [], 
-  "created": "1995-07-31T13:45:00", 
-  "creators": [
-    "test_user_1_"
-  ], 
-  "description": "", 
-  "effective": null, 
-  "exclude_from_nav": false, 
-  "expires": null, 
-  "id": "document", 
-  "is_folderish": false, 
-  "language": "", 
-  "layout": "document_view", 
-  "lock": {
-    "created": "1995-07-31T17:30:00", 
-    "creator": "admin", 
-    "creator_name": "admin", 
-    "creator_url": "http://localhost:55001/plone/author/admin", 
-    "locked": true, 
-    "name": "iterate.lock", 
-    "stealable": false, 
-    "time": 807201000.0, 
-    "timeout": 4294967280, 
-    "token": "0.3217605825198149-0.016714612599692202-00105A989226:1629895144.744"
-  }, 
-  "modified": "1995-07-31T17:30:00", 
-  "next_item": {
-    "@id": "http://localhost:55001/plone/copy_of_document", 
-    "@type": "Document", 
-    "description": "", 
-    "title": "Test document"
-  }, 
-  "parent": {
-    "@id": "http://localhost:55001/plone", 
-    "@type": "Plone Site", 
-    "description": "", 
-    "title": "Plone site"
-  }, 
-  "previous_item": {}, 
-  "relatedItems": [], 
-  "review_state": "private", 
-  "rights": "", 
-  "subjects": [], 
-  "table_of_contents": null, 
-  "text": null, 
-  "title": "Test document", 
-  "version": "current", 
-  "working_copy": {
-    "@id": "http://localhost:55001/plone/copy_of_document", 
-    "created": "1995-07-31T13:45:00", 
-    "creator_name": "admin", 
-    "creator_url": "http://localhost:55001/plone/author/admin", 
-    "title": "Test document"
-  }, 
-  "working_copy_of": null
-}
\ No newline at end of file
+    "@components": {
+        "actions": {
+            "@id": "http://localhost:55001/plone/document/@actions"
+        },
+        "breadcrumbs": {
+            "@id": "http://localhost:55001/plone/document/@breadcrumbs"
+        },
+        "contextnavigation": {
+            "@id": "http://localhost:55001/plone/document/@contextnavigation"
+        },
+        "navigation": {
+            "@id": "http://localhost:55001/plone/document/@navigation"
+        },
+        "types": {
+            "@id": "http://localhost:55001/plone/document/@types"
+        },
+        "workflow": {
+            "@id": "http://localhost:55001/plone/document/@workflow"
+        }
+    },
+    "@id": "http://localhost:55001/plone/document",
+    "@type": "Document",
+    "UID": "SomeUUID000000000000000000000001",
+    "allow_discussion": false,
+    "contributors": [],
+    "created": "1995-07-31T13:45:00",
+    "creators": [
+        "test_user_1_"
+    ],
+    "description": "",
+    "effective": null,
+    "exclude_from_nav": false,
+    "expires": null,
+    "id": "document",
+    "is_folderish": false,
+    "language": "",
+    "layout": "document_view",
+    "lock": {
+        "created": "1995-07-31T17:30:00",
```

Comment:
Funny thing is that locally for me, only this line got different. She stayed:

```
        "created": "1995-07-31T14:30:00",
```

It seems that only in this part the TZ environment variable had no effect.

Reply:
humm this is explained here:

https://github.com/plone/plone.restapi/pull/1215/commits/3a4e8f3a421accd18e5eb2927e441289bdced5bc

@mauritsvanrees wouldn't it be the case to normalize this too, so as not to show differences locally?

Reply:
Possibly, but I wonder why the timezone has no effect here.
@avoinea This is the created date of a lock. Can you run the tests locally and then check if this results in a local change in git?

Reply:
It's really weird that all dates respect the `TZ` environment variable and only this date doesn't.

I saw that who sets this date is the function:

https://github.com/plone/plone.restapi/blob/df3dba9b2eff44dc6349175ffdd9e97157ac58a9/src/plone/restapi/services/locking/__init__.py#L18-L19

Maybe this can help:

https://stackoverflow.com/questions/31977563/python-how-do-you-convert-a-datetime-timestamp-from-one-timezone-to-another-tim

@avoinea FYI

Reply:
@avelino @mauritsvanrees 

@ericof fell in this hole too:

https://github.com/plone/plone.restapi/commit/2bc6910f3cf8bc856c6138d8e1828e7d87b4c765